src/Management/src/Endpoint/CloudFoundry/CloudFoundrySecurityStartupFilter.cs

```diff
@@ -1,21 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the Apache 2.0 License.
-// See the LICENSE file in the project root for more information.
-
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-
-namespace Steeltoe.Management.Endpoint.CloudFoundry;
-
-public class CloudFoundrySecurityStartupFilter : IStartupFilter
-{
-    public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
-    {
-        return app =>
-        {
-            app.UseCloudFoundrySecurity();
```

Comment:
The `AllActuatorsStartupFilter ` provides the same functionality. This was removed in a previous PR but the file was actually removed because I saw that it wasn't actually being used anywhere.